<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>714</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Visualización de datos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget_reviews">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Etiqueta</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre Fichero</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sentimiento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Polaridad sentimiento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Subjetividad sentimiento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor compound sentimiento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Texto</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_filtro">
        <property name="text">
         <string>Filtro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_filtro">
        <item>
         <property name="text">
          <string>contiene</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>no contiene</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>acaba en</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>no acaba en</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>coinciden minMAYUS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>coincide exacto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>empieza por</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_filtro"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_applyfilter">
        <property name="text">
         <string>Aplicar filtro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cleanfilter">
        <property name="text">
         <string>Limpiar filtro</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_back">
      <property name="text">
       <string>Atrás</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QWidget" name="widget_resultados" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
